### ISSUE TITLE
Create "business card" option alongside profiles

### ISSUE THREAD

**Issue by newbold** [neatnik/omg.lol#128]

**Comment by lxjv** [comment]
@newbold ;) https://discord.com/channels/@me/1016741490216677458/1024715032602353765

**Comment by Vukkyy** [comment]
> [discord.com/channels/@me/1016741490216677458/1024715032602353765](https://discord.com/channels/@me/1016741490216677458/1024715032602353765)
now that's a useful link to find out more about this feature that is being developed

i can't read it!

**Comment by newbold** [comment]
@Vukkyy When I have time, I'll write up a little more about the feature and will share here. ☺️

**Comment by Catnatsuki** [comment]
:p what was this about?
a landscape orientation of the profile page?

**Comment by newbold** [comment]
Yeah, I think I was originally considering a totally different "mode" for this, but in reality it could be accomplished with a theme.
